src/main/kotlin/lotto/usecase/LottoMachine.kt

```diff
@@ -0,0 +1,23 @@
+package lotto.usecase
+
+import lotto.domain.Lotto
+
+class LottoMachine(
+    private val lottoNumberGenerator: LottoNumberGenerator,
+) {
+
+    fun buy(purchaseAmount: Int): List<Lotto> {
+        val numberOfPurchase = purchaseAmount / LOTTO_PRICE
+
+        return (1..numberOfPurchase).map {
+            Lotto(
+                numbers = lottoNumberGenerator.generate(),
+                price = LOTTO_PRICE,
+            )
+        }
+    }
+
+    companion object {
+        const val LOTTO_PRICE = 1000
```

Comment:
```suggestion
        const val LOTTO_PRICE = 1_000
```
천단위 자리에 언더바를 붙이면 가독성이 좋아집니다

Reply:
수정하였습니다.

**0c14b424**